test/EndToEnd/GivenWindowsApp.cs

```diff
@@ -32,6 +34,8 @@ public void ItCanBuildAndRun(string targetPlatformVersion)
             //  Update TargetFramework to the right version of .NET Core
             project.Root.Element(ns + "PropertyGroup")
                 .Add(new XElement(ns + "TargetPlatformVersion", targetPlatformVersion));
+            project.Root.Element(ns + "PropertyGroup")
+                .Element(ns + "TargetFramework").Value = TestAssetInfo.currentTfm;
```

Comment:
This should probably be on the list of changes to eventually revert, if it's not already.